ts/Data/Stores/GoogleDataStore.ts

```diff
@@ -0,0 +1,331 @@
+        const store = this,
+            {
+                startColumn,
+                endColumn,
+                startRow,
+                endRow
+            } = store.options,
+            columns: Array<Array<DataValueType>> = [],
+            cells = json.feed.entry,
+            cellCount = (cells || []).length;
+
+        let cell,
+            colCount = 0,
+            rowCount = 0,
+            val,
+            gr,
+            gc,
+            cellInner,
+            i: number,
+            j: number;
+
+        // First, find the total number of columns and rows that
+        // are actually filled with data
+        for (i = 0; i < cellCount; i++) {
+            cell = cells[i];
+            colCount = Math.max(colCount, cell.gs$cell.col);
+            rowCount = Math.max(rowCount, cell.gs$cell.row);
+        }
+
+        // Set up arrays containing the column data
+        for (i = 0; i < colCount; i++) {
+            if (i >= startColumn && i <= endColumn) {
+                // Create new columns with the length of either
+                // end-start or rowCount
+                columns[i - startColumn] = [];
+            }
+        }
+
+        // Loop over the cells and assign the value to the right
+        // place in the column arrays
+        for (i = 0; i < cellCount; i++) {
+            cell = cells[i];
+            gr = cell.gs$cell.row - 1; // rows start at 1
+            gc = cell.gs$cell.col - 1; // columns start at 1
+
+            // If both row and col falls inside start and end set the
+            // transposed cell value in the newly created columns
+            if (gc >= startColumn && gc <= endColumn &&
+                gr >= startRow && gr <= endRow) {
+
+                cellInner = cell.gs$cell || cell.content;
+
+                val = null;
+
+                if (cellInner.numericValue) {
+                    if (cellInner.$t.indexOf('/') >= 0 ||
+                        cellInner.$t.indexOf('-') >= 0) {
+                        // This is a date - for future reference.
+                        val = cellInner.$t;
+                    } else if (cellInner.$t.indexOf('%') > 0) {
+                        // Percentage
+                        val = parseFloat(cellInner.numericValue) * 100;
+                    } else {
+                        val = parseFloat(cellInner.numericValue);
+                    }
+                } else if (cellInner.$t && cellInner.$t.length) {
+                    val = cellInner.$t;
+                }
+
+                columns[gc - startColumn][gr - startRow] = val;
+            }
+        }
+
+        // Insert null for empty spreadsheet cells (#5298)
+        for (i = 0; i < colCount; i++) {
+            for (j = 0; j < cellCount; i++) {
+                if (typeof columns[i][j] === 'undefined') {
+                    columns[i][j] = null as any;
+                }
+            }
+        }
```

Comment:
This loop seems to be causing some issues: 
![image](https://user-images.githubusercontent.com/7478772/90229234-dd802b80-de17-11ea-9872-ac31c9eebe3e.png).

This works:
```js
	columns.forEach(function (
            column: Array<DataValueType>
        ): void {
            if(column.length){
                for (i = 0; i < column.length; i++) {
                    if (typeof column[i] === 'undefined') {
                        column[i] = null as any;
                    }
                }
            }
        });
```
Which makes it seem that we are adding additional columns somewhere.

Reply:
Maybe `colCount` is set to early?

Reply:
Will check it